integration/kubernetes/helm-chart/alluxio/templates/config/alluxio-conf.yaml

```diff
@@ -188,13 +202,14 @@ data:
   ALLUXIO_JOB_WORKER_JAVA_OPTS: |-
     {{- /* Format ALLUXIO_JOB_WORKER_JAVA_OPTS list to one line */}}
     {{ range $key := $jobWorkerJavaOpts }}{{ printf "%v " $key }}{{ end }}
+  {{- if ne .Values.worker.fuseEnabled true }}
   ALLUXIO_FUSE_JAVA_OPTS: |-
     {{- /* Format ALLUXIO_FUSE_JAVA_OPTS list to one line */}}
     {{ range $key := $fuseJavaOpts }}{{ printf "%v " $key }}{{ end }}
+  {{- end }}
   ALLUXIO_WORKER_TIEREDSTORE_LEVEL0_DIRS_PATH: /dev/shm
   {{- if .Values.logserver.enabled }}
   ALLUXIO_LOGSERVER_HOSTNAME: {{ $fullName }}-logserver
   {{- /* Note: The value must be a string or kubectl will fail to read it */}}
   ALLUXIO_LOGSERVER_PORT: {{ quote .Values.logserver.ports.logging }}
   {{- end }}
-
```

Comment:
this is the end of file line? every file should have an empty line at the end of the file

Reply:
Isn't this because gitHub doesn't show the empty line at the end? I can see the empty line in ItelliJ.